fighters/peach/src/status/attack_air.rs

```diff
@@ -92,6 +92,8 @@ unsafe extern "C" fn peach_attack_air_no_float_main_loop(fighter: &mut L2CFighte
 
     
     if fighter.global_table[CURRENT_FRAME].get_i32() <= ParamModule::get_int(fighter.battle_object, ParamType::Common, "djc_leniency_frame")
+    && fighter.global_table[CURRENT_FRAME].get_i32() > 1
+    && !StopModule::is_stop(fighter.module_accessor)
```

Comment:
`!fighter.is_in_hitlag` pls

Reply:
im sorry i didn't pull before pushing and it did that